src/pages/studyView/table/MultiSelectionTable.tsx

```diff
@@ -54,7 +54,7 @@ export type MultiSelectionTableRow = OncokbCancerGene & {
 
 export enum MultiSelectionTableColumnKey {
     GENE = 'Gene',
-    MOLECULAR_PROFILE = 'Molecular Profile',
```

Comment:
@viveak910 this seems risky.  the enum is named "...key" which suggests perhaps it is used for logical purposes somewhere.   If in fact it is only a label, lets rename the enum